workflows/amr/run.wdl

```diff
@@ -616,31 +616,70 @@ task RunSpades {
         String docker_image_id
     }
     command <<< 
-        set -euxo pipefail
-        function handle_failure() 
-        {
-            echo ";ASSEMBLY FAILED" > spades/contigs.fasta
-            echo ";ASSEMBLY FAILED" > spades/scaffolds.fasta
-            exit 0
+    set -euxo pipefail
+    function handle_failure()
+    {
+        echo ";ASSEMBLY FAILED" > spades/contigs.fasta
+        echo ";ASSEMBLY FAILED" > spades/scaffolds.fasta
 
-        }
-        trap handle_failure ERR
-        if [[ "~{length(reduplicated_reads)}" -gt 1 ]]; then
-            spades.py -1 ~{sep=" -2 " reduplicated_reads} -o "spades/" -m 100 -t 36 --only-assembler 1>&2
-        else
-            spades.py -s ~{reduplicated_reads[0]} -o "spades/" -m 100 -t 36 --only-assembler 1>&2
-        fi
+        python3 <<CODE
+    import json
+    import os
+
+    WARNINGS_LOG = "spades/warnings.log"
+    SPADES_LOG = "spades/spades.log"
+
```

Comment:
Could we reuse the script above?

Reply:
I can try again. I ran into an issue where modifying the RunSpades task to rely on a change in the short-read-mngs container broke several AMR tests. I don't remember if this prevented a build and push of the new image, I will look into it again.